src/TheNextWeek/texture.h

```diff
@@ -58,7 +58,8 @@ class noise_texture : public texture {
         virtual vec3 value(double u, double v, const vec3& p) const {
             // return vec3(1,1,1)*0.5*(1 + noise.turb(scale * p));
             // return vec3(1,1,1)*noise.turb(scale * p);
-            return vec3(1,1,1)*0.5*(1 + sin(scale*p.x() + 5*noise.turb(scale*p)));
+            return vec3(1,1,1)*
+                0.5 * (1 + sin(scale*p.z() + 10*noise.turb(p)));
```

Comment:
Now that we've standardized on 96-characters per line, you can put this on a single line.